drivers/scheduler/k8s/specs/cassandra/cassandra-stress.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cassandra-stress
+  labels:
+    app: cassandra-stress
+spec:
+  replicas: 3
```

Comment:
why do we need 3 replicas for cassandra-stress? I think it should be one instance of cassandra-stress which will run some load using cassandra service endpoint

Reply:
addressed these comments in #556